### PR TITLE
add amazon inspector agent

### PR DESCRIPTION
**Pull request by edulop91**
also moving to t3.mediums to increase iteration speed

### CODE REVIEW

File: packer/ansible/roles/harden/tasks/inspector.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- name: download inspector
+  get_url:
+    url: https://inspector-agent.amazonaws.com/linux/latest/install
```
Comment (ryanking) [inline packer/ansible/roles/harden/tasks/inspector.yml]: do we want to pin to a version? at least a major version?

Comment (edulop91) [reply packer/ansible/roles/harden/tasks/inspector.yml]: https://docs.aws.amazon.com/inspector/latest/userguide/inspector_agents.html#agent-updates

Comment (edulop91) [reply packer/ansible/roles/harden/tasks/inspector.yml]: i have not been able to find an option to download specific versions. looks like the agent auto-updates itself. bc of auto-updates don't think pinning a version would really do much for us

File: packer/ansible/harden.yml [hunk 2]
```diff
@@ -2,21 +2,21 @@
   become: yes
   vars:
     # https://docs.openstack.org/ansible-hardening/latest/rhel7/auto_controls-by-severity.html
-    security_check_package_checksums: yes # generating checksums is slow
-    security_enable_gpgcheck_repo: yes # V-71981
```
Comment (edulop91) [inline packer/ansible/harden.yml]: fails on CentOS - al2 is a centos derivative

**Pull request merged**